planet4-gutenberg-blocks.php

```diff
@@ -346,6 +346,19 @@ function ( $posts, $query ) {
 
 add_filter( 'timber/twig', 'p4_blocks_en_forms_twig_filters' );
 
+/**
+ * WordPress automatically adds rel="noopener" to links that have _blank target.
+ * This function removes that functionality since it breaks SocialMedia WYSIWYG blocks with Instagram embeds.
+ *
+ * @param string $rel The link rel.
+ * @param string $link The link.
+ * @return boolean
+ */
+function remove_automatic_link_rel( $rel, $link ) {
```

Comment:
This is the [solution](https://github.com/WordPress/gutenberg/issues/9731#issuecomment-475382493) I've found to solve the block validation failing with Instagram embed, but I'm not sure that it's the right call so any suggestion is welcome 😬